src/app/profile/email-verification/email-verification.component.html

```diff
@@ -0,0 +1,44 @@
+<div  class="container f8-email-verify">
+  <div class="row">
+    <div class="blank-slate-pf">
+      <div class="rh-branding-logo">
+        <img class="width-100"
+          src="../../../assets/images/Logotype_RH_OpenShift-io_RGB_RedGray.png">
+      </div>
+      <div *ngIf="verified === 'true'">
+        <img 
+          width="100"
+          height="100" 
+          src="../../../assets/images/trophy.png" />
+        <h1>
+          Your e-mail has been confirmed
+        </h1>
+        <hr>
+        <p>
+          Thank you for validating your e-mail address.
+          You can now continue to use Openshift.io
+        </p>
+      </div>
+      <div *ngIf="verified === 'false'">
+        <img
+          width="100"
+          height="100"
+          src="../../../assets/images/neutralface.png">
+        <h1>
+          This URL has already been used to verify your e-mail
```

Comment:
The actuall error message is passed as a parameter too:
If failed - http://redirect.url?verified=false&error=ERROR_MESSAGE
If successful then just - http://redirect.url?verified=true

Reply:
@catrobson @mindreeper2420  I'm guessing the actual error message wasn't considered in the UX-provider design? :)
Could we display the error message that was passed back in a nice way somewhere?

Reply:
@alexeykazakov @catrobson How we are displaying the screens with error messages?